causal_testing/json_front/json_class.py

```diff
@@ -246,7 +250,8 @@ def _populate_metas(self):
             meta.populate(self.data)
 
     def _execute_test_case(
-        self, causal_test_case: CausalTestCase, test: Iterable[Mapping], f_flag: bool
+            self, causal_test_case: CausalTestCase, test: Iterable[Mapping],
```

Comment:
Just noticed this, but why is `test` and `Iterable`?

Reply:
Good spot, it should just be Mapping